monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -58,7 +58,7 @@ class ConfigurePageComponent extends AuthComponent {
   }
 
   getSectionsOrder() {
-    let islandMode = this.props.islandMode ? this.props.islandMode : 'advanced'
+    let islandMode = 'advanced'
```

Comment:
I'm guessing this was done for testing and committed accidentally?

Reply:
Exactly. It will be put back.